internal/build/templates/ensure-sysext.service

```diff
@@ -9,12 +9,16 @@ ConditionDirectoryNotEmpty=|/run/extensions
 ConditionDirectoryNotEmpty=|/var/lib/extensions
 ConditionDirectoryNotEmpty=|/usr/local/lib/extensions
 ConditionDirectoryNotEmpty=|/usr/lib/extensions
+ConditionPathExists=/usr/bin/elemental3ctl
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=/usr/bin/systemctl daemon-reload
+# TODO (atanasdinov): Investigate a SELinux limitation where elemental3ctl cannot be called directly here
+ExecStart=/bin/sh -c "/usr/bin/elemental3ctl kmod --reload"
 ExecStart=/usr/bin/systemctl restart --no-block sockets.target timers.target multi-user.target
+ExecStop=/bin/sh -c "/usr/bin/elemental3ctl kmod --unload"
```

Comment:
I'm torn a little on this; do we explicitly want to unload modules without rebooting into the *new* ones (or *older* ones if it's a rollback)? I'm trying to figure out the ideal UX here, whether it makes sense to only load, and assume that any reload operations are part of a reboot, where we also refresh the OS image. What use-case(s) do I need to be thinking about here?